### DISCUSSION TITLE
Custom sorting function

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#395] [Feedback & Feature Proposal]
Following https://github.com/meilisearch/meilisearch/discussions/2165 and the usecase described, the users might need the possibility to create their own sorting functions.

I discussed this with @Kerollmops and we might encounter some technical complications, so we'll need to define what is technically doable or not

**Comment by sandstrom** [comment]
This seems like a duplicate of https://github.com/meilisearch/product/discussions/205.

For anyone who's interested in this functionality, upvote [this issue too](https://github.com/meilisearch/product/discussions/205).

**Comment by anhle199** [comment]
Hi @curquiza, are there any updates about this feature?
